jinahub/indexers/storage/RedisStorage/manifest.yml

```diff
@@ -0,0 +1,9 @@
+manifest_version: 1
+name: RedisStorage
+alias: RedisStorage
```

Comment:
make sure you've pushed it to Hubble and added the secrets to the secrets location (check Jina Hub channel)